Workflow automation/Get-SecureScoreData/Grant-SubscriptionPermissions.ps1

```diff
@@ -0,0 +1,9 @@
+Connect-AzAccount
```

Comment:
Nick, I suggest to open new folder in the main GitHub folder called "Secure score" that will contain this and also Amit's PowerBI dashboard in the future - as this isn't only workflow automation playbooks. WDYT?

Reply:
Update pull #116 to include folder path Secure score and move Send-SecureScoreBriefing logic app to new location for other assets - https://github.com/Azure/Azure-Security-Center/pull/116

Reply:
Created new folder.  @swiftsolves-msft will need to move his logic app.